### ISSUE TITLE
Y axis width control on all x=time panels

### ISSUE THREAD

**Issue by leeoniya** [grafana/grafana#85040]
some panels dont support y axis width control via override, which makes them impossible to align vertically with ones that do.

StatusHistory
StateTimeline
Heatmap

StatusHistory is a bit different cause it grows its visible time range to fit the edge cells, so we should add control for allowing them to be cut in half as well? not great, but maybe.

**Comment by Strunck** [comment]
Happy to see some movement.
Meanwhile I would like to make a proposal. For multiple panels with the same Y-Axis, the information conveyed would be redundant. 
How about this?
Only one Y-Axis for two different panels.
![Shared Y-Axis for multiple chartareas](https://github.com/grafana/grafana/assets/64785859/b2f6e8c2-ed5f-4590-a1b4-6020d7fc6b12)


This is, how it looked before:
![125912128-cfa1a035-fa11-44d2-be4d-f28682170638](https://github.com/grafana/grafana/assets/64785859/f8c1f58a-8439-4a13-b681-90e2322281cc)

**Comment by Strunck** [comment]
This is a grafana screenshot:
![Other example spreaded](https://github.com/grafana/grafana/assets/64785859/fb057a19-2543-4826-b335-48299b803f8e)

In Windows Paint I reduced the Y-Axis.
![Other example one Y](https://github.com/grafana/grafana/assets/64785859/42751ee3-85eb-4de6-bd56-e39bdffae8e4)
It is just a detail, but it makes it more easy to understand. And the is no Y-Axis redundancy.

**Comment by nabelekt** [comment]
Agreed that this is needed. I am trying to do it in state timelines. The misalignment makes it hard to analyze timing across the timelines.
![Image](https://github.com/user-attachments/assets/2bb285a6-d734-4087-bd13-942fb90c3819)

**Comment by nabelekt** [comment]
Remove stale
